packages/facility-language-server/package.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: facility-language-server
+description: |
+  Facility Service Definition language. This version of Facility Service Definition
+  requires dotnet (.NET 6.0) to be installed.
+homepage: https://facilityapi.github.io/
+licenses:
+  - MIT
+languages:
+  - Facility Service Definition
+categories:
+  - LSP
+
+source:
+  id: pkg:github/FacilityApi/FacilityLanguageServer@v2.1.0
+  asset:
+    - target: darwin_x64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: darwin_arm64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: win_x64
+      file: Facility.LanguageServer.zip:libexec/
```

Comment:
Should this not be `Facility.LanguageServer-win-x64.zip`? Also is there no Linux support?

Reply:
`Facility.LanguageServer.zip` is platform neutral. `Facility.LanguageServer-win-x64.zip` contains an exe to make running on Windows easier for the Visual Studio extension.